### PR TITLE
JDK-11 support: refactor code to avoid underscore variable names

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Refactoring towards Java 11. Underscore variables are not welcome in Java anymore, and IMO it makes sense to just refactor them away.

# Changelog

* Not required

@jenkinsci/sig-platform

### CODE REVIEW

**Review by dwnusbaum — APPROVED** [review]

**Review by slide — APPROVED** [review]

File: core/src/main/java/hudson/ClassicPluginStrategy.java [hunk 1]
```diff
@@ -788,7 +788,7 @@ private void updateTransientDependencies() {
                     if(p!=null)
                         try {
                             return p.classLoader.loadClass(name);
-                        } catch (ClassNotFoundException _) {
+                        } catch (ClassNotFoundException ex) {
```
Comment (daniel-beck) [inline core/src/main/java/hudson/ClassicPluginStrategy.java]: Would prefer `ignored` so IntelliJ understands it.

**Review by MarkEWaite — APPROVED** [review]
I'm OK with the change as is, or `ignored` would also be fine (as suggested by @daniel-beck

**Comment by oleg-nenashev** [comment]
@daniel-beck will fix it this week

**Review by johnou — APPROVED** [review]

**Review by MarkEWaite — APPROVED** [review]

**Review by johnou — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Going to merge, got into a merge conflict in a follow-up PR. I believe the feedback is addressed, happy to do any kind of follow-up refactoring if needed

**Pull request merged**